static/js/directives/report-image.js

```diff
@@ -0,0 +1,16 @@
+angular.module('inboxDirectives').directive('reportImage',
+  function(DB) {
```

Comment:
If you're going to do DI this way you need to add the `"ngInject"` directive. We also use `"use strict"` everywhere. so add:

```
  'use strict';
  'ngInject';
```